scen_edit/command/sync/team_sync.lua

```diff
@@ -82,11 +82,6 @@ function TeamManagerListenerWidget:onTeamAdded(teamID)
         return
     end
 
-    -- get local AI name (synced has weird AI names)
```

Comment:
Can you elaborate the changes here and in team_manager.lua? How is this an improvement? (Asking because it doesn't seem obvious to me - not criticizing.)